frappe/core/doctype/view_log/view_log.json

```diff
@@ -124,7 +124,7 @@
  "modified": "2018-05-30 11:21:35.258019", 
  "modified_by": "shridhar.p@zerodha.com", 
  "module": "Core", 
- "name": "View log", 
+ "name": "View Log", 
```

Comment:
Did you manually change this in code, or was this from editing the DocType?

I'm not sure if this will take effect without a change in the `modified` timestamp.

Reply:
@Alchez renaming the doctype failed.. 
this will update though, because we're calling reload_doc in the patch.

Reply:
What about temporarily renaming to a random name, and then moving back to "View Log"?

Also, the patch won't run for new site installs. Will this still work then?

Reply:
Hey, I tried that too.. didn't work with another error..
The patch wouldn't be required for new installs ;)

Reply:
Oh, cool. Thanks for the replies :)

Reply:
Thanks for the questions, it is much appreciated :)